lib/vquic/msh3.c

```diff
@@ -95,7 +95,7 @@ static const MSH3_REQUEST_IF msh3_request_if = {
 
 void Curl_quic_ver(char *p, size_t len)
 {
-  (void)msnprintf(p, len, "msh3/%s", "0.0.1");
+  (void)msnprintf(p, len, "msh3/%s", "0.3.0");
```

Comment:
This seems brittle, can we persuade the good msh3 folks to add version information to `msh3.h` which we can consume?

Reply:
I agree! That's why I created https://github.com/nibanks/msh3/issues/23. Feel free to take a crack at it.

Reply:
RIght. it's useful if version string API in mash3. @nibanks What do you think about?

Reply:
As https://github.com/nibanks/msh3/pull/38 has been merged now, this can now get the runtime version!

Reply:
Ok! I'll try to it.

Reply:
@bagder I addressed it at deb1bc3

![image](https://user-images.githubusercontent.com/4487291/166082000-eef15833-ebcf-4eef-b290-64e6585f8c6d.png)

Reply:
LGTM!